### PR TITLE
Template Imports and Exports

### PR DESCRIPTION
**Pull request by christophertubbs**
Added the ability to import and export evaluation templates in the evaluation service and added stock templates available to all evaluation service instances.

### CODE REVIEW

**Review by robertbartel — COMMENTED** [review]
Ok, I've reviewed these changes.  I wouldn't mind having another set of eyes on it, as this is rather large and I'm not as familiar with the evaluation code as I am with other parts.   But if no one has time after a couple more days, I'll come back and approve it.

Remind me @christophertubbs: are those additional tests you'd been developing relatively recently (certainly compared to this) related to this work, or something else?  There do seem to be a few pieces being added without modifications/additions to the tests.   We can live with it, but it might be good to open an issue for later.

**Comment by christophertubbs** [comment]
> Remind me @christophertubbs: are those additional tests you'd been developing relatively recently (certainly compared to this) related to this work, or something else? There do seem to be a few pieces being added without modifications/additions to the tests. We can live with it, but it might be good to open an issue for later.

Unit tests are for bootstrapping for confidence intervals - not this.

**Comment by robertbartel** [comment]
@christophertubbs, if you'll resolve these conflicts (and assuming tests pass after), I'll go ahead and approve this.

**Review by robertbartel — APPROVED** [review]

**Pull request merged**
